src/super_gradients/common/environment/environment_config.py

```diff
@@ -1,15 +1,38 @@
+import dataclasses
 import os
-
+import argparse
 import pkg_resources
 
+__all__ = ["PKG_CHECKPOINTS_DIR", "EXTRA_ARGS", "device_config"]
+
+
 try:
     PKG_CHECKPOINTS_DIR = pkg_resources.resource_filename("checkpoints", "")
 except Exception:
     os.makedirs(os.path.join(os.getcwd(), "checkpoints"), exist_ok=True)
     PKG_CHECKPOINTS_DIR = os.path.join(os.getcwd(), "checkpoints")
 
 
-DDP_LOCAL_RANK = int(os.getenv("LOCAL_RANK", default=-1))
+def _get_requested_rank():
```

Comment:
May I suggest to rename the function and corespodnding variable name from `requested_rank` to `assigned_rank`? I feel it would have better clarity on what this functon returns.